packages/app-project/pages/404.js

```diff
@@ -1,14 +1,17 @@
+import { ThemeContext } from 'grommet'
 import { ZooHeaderWrapper } from '@components'
 import { ZooFooter } from '@zooniverse/react-components'
 import { Box, Image, Heading, Text, Paragraph, Anchor } from 'grommet'
 import { serverSideTranslations } from 'next-i18next/serverSideTranslations'
 import { useTranslation } from 'next-i18next'
+import theme404 from './../src/helpers/404theme.js'
 import styled from 'styled-components'
 
 
 const ContainerBox = styled(Box)`
   position: relative;
   color: white;
+	background-color: #000;
```

Comment:
ContainerBox still has a `background` prop. It appears here and is set to '#1e1e1e'.
![Screenshot 2023-09-22 at 10 40 19 AM](https://github.com/zooniverse/front-end-monorepo/assets/23665803/d279e298-c931-4611-8f37-ef8b21484763)

By moving `background-color` to the styled-component function, this element still has two background colors defined and '#1e1e1e' is being overwritten by '#000':
<img width="775" alt="Screenshot 2023-09-22 at 10 39 21 AM" src="https://github.com/zooniverse/front-end-monorepo/assets/23665803/c423da86-b745-422b-8893-32d8a619f2fb">

Apologies if you're still intending to address my initial comment about the `background` prop, but hope this better explains why the `background` prop should be removed from ContainerBox in favor of defining `background-color` as black once.